bridges/snowbridge/pallets/ethereum-client/src/benchmarking/mod.rs

```diff
@@ -111,13 +113,43 @@ mod benchmarks {
 		let update = make_sync_committee_update();
 		let block_root: H256 = update.finalized_header.hash_tree_root().unwrap();
 
+		let fork_versions = ForkVersions {
+			genesis: Fork {
+				version: [0, 0, 0, 0], // 0x00000000
```

Comment:
It seems these fork versions are [canonically](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/fork.md#configuration) defined using hex. To reduce the chance of trancription errors, why not just do the same here?

```suggestion
				version: hex!("00000000")
```

Reply:
That's a good point, https://github.com/paritytech/polkadot-sdk/pull/7075/commits/f758355a3a808078f97ac88b00fa26bccbccbcff.